tests/foreman/ui/test_computeresource_gce.py

```diff
@@ -32,9 +32,7 @@
 @pytest.mark.tier2
 @pytest.mark.upgrade
 @pytest.mark.skip_if_not_set('http_proxy', 'gce')
-def test_positive_default_end_to_end_with_custom_profile(
-    session, module_org, module_loc, module_gce_settings
```

Comment:
is there a reason why `module_gce_settings` is removed ?

Reply:
The fixture doesn't exist